frontend/src/toolbar/actions/actionsTabLogic.tsx

```diff
@@ -22,14 +22,14 @@ function newAction(element: HTMLElement | null, dataAttributes: string[] = []):
 function toElementsChain(element: HTMLElement): ElementType[] {
     const chain: HTMLElement[] = []
     let currentElement: HTMLElement | null | undefined = element
-    while (currentElement && chain.length <= 10 && currentElement !== document.body) {
+    while (currentElement && currentElement !== document.body.parentNode) {
         chain.push(currentElement)
         currentElement = currentElement.parentElement
     }
     return chain.map(
         (element, index) =>
             ({
-                attr_class: element.getAttribute('class') || undefined,
```

Comment:
`{} satisfies ElementType` catches this error. I wanted to use it, but had the following issues with CI:
* [_Code quality checks > Generate logic types and run typescript with strict_](https://github.com/PostHog/posthog/actions/runs/5306405799/jobs/9603965244?pr=16126)
* [_Build PostHog > Build container images_](https://github.com/PostHog/posthog/actions/runs/5306405805/jobs/9603965246?pr=16126)

Reply:
> Error: did not recognize object of type "TSSatisfiesExpression"

@mariusandra is this something that needs fixing in Kea TypeGen?